srcpkgs/llhttp/files/Makefile

```diff
@@ -0,0 +1,25 @@
+# Makefile template for shared library
+
+CC ?= gcc 
```

Comment:
problably not set it and let us set it ?

Reply:
Uh? `?=` means default, if you set CC it'll be overruled

Reply:
ah ok